src/03-Parent-Child-Relationships/ReadMe.md

```diff
@@ -146,12 +146,12 @@ Let's say a parent-like component has one child-like component. Answers to the f
 - With which child in particular are we communicating?
 
 However, when a parent has two or more children, we cannot give obvious answers to the questions above. Rather, complicated situations can arise.
-- **The "Slot Query" problem:** Two or more children may use diffrent `query` types, so which `query` type's value do we use when communicating with Child A instead of Child B?
+- **The "Slot Query" problem:** Two or more children may use different `query` types, so which `query` type's value do we use when communicating with Child A instead of Child B?
 - **The "Slot Message" problem:** Two or more children may use different `message` types to raise/emit messages to the parent. How does the parent correctly map each one to the parent component's `action` type?
 - **The "Slot Index" problem:** Two or more children may use the `same` query type, so which of those children do we query?
 - **The "Slot Label" problem:** The above three problems allow all sorts of different combinations, so how does the parent track one combination from another?
 
-For languages that don't have a powerful type checker, these kinds of problems can lead to runtime errors and tracking down the bugs become difficult.
+For languages that don't have a powerful type checker, these kinds of problems can lead to runtime errors and tracking down the bugs becomes difficult.
```

Comment:
I think the original rendering is actually correct once you remove the other clauses:
> these kinds of problems ... become difficult.

Reply:
I'm pretty confident that my fix is correct, because I'm parsing the sentence as this:

> these kinds of problems (plural) can lead (<- this verb is connected to "problems") to runtime errors and tracking down the bugs becomes (<-this verb is connected to "tracking down" = singular) difficult.

To be honest I can't imagine parsing of this sentence where `become` is connected to `these kinds of problems`. But if you insist, I'll change it back.